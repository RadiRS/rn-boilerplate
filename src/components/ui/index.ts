export { default as Text } from './text';
export { default as Button } from './button';
