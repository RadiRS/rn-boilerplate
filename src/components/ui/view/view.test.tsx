import React from 'react';
import { render } from '@testing-library/react-native';

import { View } from '@/components/ui';

describe('<View />', () => {
  it('should render component correctly', () => {
    const rendered = render(<View />).toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
