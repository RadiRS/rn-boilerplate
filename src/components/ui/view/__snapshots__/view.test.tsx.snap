// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<View /> should render component correctly 1`] = `<View />`;
